spec/support/acceptance/helpers.rb

```diff
@@ -0,0 +1,116 @@
+require 'puppet_litmus'
+# rubocop:disable Style/BracesAroundHashParameters
+
+# The Target class and TargetHelpers module are a useful ways
+# for tests to reuse Litmus' helpers when they want to do stuff
+# on nodes that may not be the current target host (like e.g.
+# the master or the ServiceNow instance).
+#
+# NOTE: The code here is Litmus' recommended approach for multi-node
+# testing (see https://github.com/puppetlabs/puppet_litmus/issues/72).
+# We should revisit it once Litmus has a standardized pattern for
+# multi-node testing.
+
+class Target
+  include PuppetLitmus
+
+  attr_reader :uri
+
+  def initialize(uri)
+    @uri = uri
+  end
+
+  # Make sure that ENV['TARGET_HOST'] is set to uri
+  # before each PuppetLitmus method call. This makes it
+  # so if we have an array of targets, say 'agents', then
+  # code like agents.each { |agent| agent.bolt_upload_file(...) }
+  # will work as expected. Otherwise if we do this in, say, the
+  # constructor, then the code will only work for the agent that
+  # most recently set the TARGET_HOST variable.
+  PuppetLitmus.instance_methods.each do |name|
```

Comment:
We probably won't need something like this, but this _is_ the right way to set `TARGET_HOST` and it isn't too hard to understand so I'd like to keep the general solution.